.github/workflows/ocr-tests.yml

```
@@ -0,0 +1,32 @@
+on:
+  pull_request:
+  push:
+    branches: [ 'sync-copilot-instructions', 'feat/ocr-runner' ]
+
+jobs:
+  ocr-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.9)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)